core/unit_test/TestMathematicalSpecialFunctions.hpp

```diff
@@ -1967,31 +1967,61 @@ TEST(TEST_CATEGORY, mathspecialfunc_errorfunc) {
 #endif
 
 TEST(TEST_CATEGORY, mathspecialfunc_cbesselj0y0) {
+#if defined(KOKKOS_ENABLE_OPENMPTARGET) && defined(KOKKOS_ARCH_INTEL_GPU)
+  if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::OpenMPTarget>)
+    GTEST_SKIP() << "skipping since test is known to fail with OpenMPTarget on "
+                    "Intel GPUs";
```

Comment:
Don't forget the FIXMEs
```suggestion
    GTEST_SKIP() << "skipping since test is known to fail with OpenMPTarget on "
                    "Intel GPUs";  // FIXME_OPENMPTARGET
```